opentracing-api/src/main/java/io/opentracing/ThreadLocalActiveSpan.java

```diff
@@ -0,0 +1,143 @@
+package io.opentracing;
+
+import java.util.Map;
+import java.util.concurrent.atomic.AtomicInteger;
+
+ /**
+ * {@link ThreadLocalActiveSpan} is a trivial {@link ActiveSpan} implementation that relies on Java's thread-local
+ * storage primitive.
+ *
+ * @see ActiveSpanSource
+ * @see Tracer#spanSource()
+ */
+public class ThreadLocalActiveSpan implements ActiveSpan {
+    private ThreadLocalActiveSpanSource source;
+    private final Span wrapped;
```

Comment:
personal preference: s/wrapped/span/, because `this.wrapped` is not informative.